### PR TITLE
linux_heap_glibc: Fix threaded arenas following

### PR DESCRIPTION
**Pull request by morbith-dqtz**

### CODE REVIEW

File: libr/core/linux_heap_glibc.c [hunk 1]
```diff
@@ -1457,6 +1457,7 @@ static void GH(print_heap_mmaped)(RCore *core, GHT malloc_state, GHT global_max_
 }
 
 void GH(print_malloc_states)( RCore *core, GHT m_arena, MallocState *main_arena) {
+	const int tcache = r_config_get_i (core->config, "dbg.glibc.tcache");
```
Comment (nsajko) [inline libr/core/linux_heap_glibc.c]: Is having a const-qualified variable initialized by a function standard C?

**Comment by radare** [comment]
Yes its fine. Const means its not gonna change after assignment. Not const at compile time

File: libr/core/linux_heap_glibc.c (hunk 1 above)
Comment (morbith-dqtz) [reply libr/core/linux_heap_glibc.c]: i'll change it, thx.

**Comment by morbith-dqtz** [comment]
I can't follow why travis failed

**Comment by nsajko** [comment]
Travis just timed out, it will probably be fine next time.

**Comment by nsajko** [comment]
Try amending the commit and force-pushing to rerun the tests.

**Comment by radare** [comment]
nobody can because there are > 50 tests that report XX] but they are ignored because of @ret2libc changes after enabling ptrace in travisland

**Comment by radare** [comment]
also, timeouts happen because travis infrastructure is not 100% reliable, so if its just 1 for timeout its usually travis fault

**Pull request merged**